account: Replace the application owner reference with org administrator (#1840)
balena-io/docs: pages/learn/manage/account.md

### Member types

| Member Type   | Add members   | Delete App  | Add/Remove device  | Device actions  | Tags | Dev Env Variables | SSH access | Push | Fleet Env Variables | Fleet actions |
| ------------- |:-------------:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:|
| Administrator | Yes      | Yes  | Yes  | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
| Developer     | No       | No   | Yes  | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
| Operator      | No       | No   | Yes  | Yes  | Yes  | Yes | Yes | No  | No  | No  |
| Observer      | No       | No   | No   | No   | No   | No  | No  | No  | No  | No  |

#### Administrator

> A new application is always created by an an [administrator][administrator] of the organization. Administrators are the only users who can add other application members or delete the application. Learn more about the [administrator role](adminstrator) in an organization's scope.

Reviewer: ```suggestion
A new application in balenaCloud can only be created by an [administrator][administrator] of an organization. Administrators are the only users who can add other application members or delete the application. Learn more about the [administrator role](adminstrator) in an organization.
```

Remove 2 `an`
Generalized it a bit and removed the scope bit.

Author: Well @vipulgupta2048 the change type squash is so that this PR doesn't get merged before I have the change to merge my commits. even in case someone approves :)

| Member Type | Add members   | Delete App  | Add/Remove device  | Device actions  | Tags | Dev Env Variables | SSH access | Push | Fleet Env Variables | Fleet actions |
| ------------- |:-------------:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:|

> | Administrator | Yes      | Yes  | Yes  | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |

Reviewer: The no longer consistent spacing of the table is triggering my OCD ... big time 😨

| Observer   | No       | No   | No   | No   | No   | No  | No  | No  | No  | No  |
| Operator   | No       | No   | No   | Yes  | Yes  | Yes | Yes | No  | No  | No  |
| Developer  | No       | No   | No   | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |

#### Administrator

> The Organization's [Administrators][orgAdministrator] are the users who first create an application. They are the only users who can add other application members or delete the application.

Reviewer: ```suggestion
In an application's scope, an [administrator][administrator] is a user who created the application. There can be multiple administrators in an application. Administrators are the only users who can add other application members or delete the application. Learn more about the [administrator's role](adminstrator) in an organization's scope. 
```

Author: Well there can obviously be administrators who have never created an app.
How about the one I just pushed?

Author: Technically we could just drop this from here, and also drop some of the columns of from the table, but part of me likes how it compares what app members can do, vs what administrators only can do.

Reviewer: Yeah true that on the first point. 
That part of you that likes "that this is on the table" is right. From an application scope, I think we should explain the role of admins and that's what this does.

[ssh]:/learn/manage/ssh-access
[starter]:/learn/manage/app-types#starter

> [orgAdministrator]: /learn/manage/organizations/#managing-roles--access-in-an-organization

Reviewer: ```suggestion
[administrator]: /learn/manage/organizations/#managing-roles--access-in-an-organization
```
Do make sure this anchor is correct
